best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use JavaScript with [babel], or another language that compiles to JavaScript.
```

Comment:
This becomes more ambiguous as it doesn't specify ES6/ES2015, which might be construed as continuing to use ES5. Should we retain explicitness around ES6/ES2015?